src/main/resources/api/openapi.yaml

```diff
@@ -532,6 +532,52 @@ components:
           allOf:
           - $ref: '#/components/schemas/ModifierId'
 
+    Parameters:
+          type: object
+          required:
+          - height
+          - K
+          - minValuePerByte
+          - maxBlockSize
+          - maxBlockCost
+          properties:
+            height:
+              type: integer
+              format: int32
+              description: Height when current parameters were considered(not actual height). Can be '0' if state is empty
+              minimum: 0
+              example: 667
+              nullable: false
+            K:
```

Comment:
@kushti I suggest to avoid such names in the API. If `K` is somehow important it can be mentioned in the description and also explained (why it is `K`?)

Reply:
@ergomorphic k renamed to storageFeeFactor